x/harvest/keeper/interest.go

```diff
@@ -2,30 +2,176 @@ package keeper
 
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	"github.com/kava-labs/kava/x/harvest/types"
 )
 
+var (
+	scalingFactor  = 1e18
+	secondsPerYear = 31536000
+)
+
 // ApplyInterestRateUpdates translates the current interest rate models from the params to the store
 func (k Keeper) ApplyInterestRateUpdates(ctx sdk.Context) {
```

Comment:
Add a comment that `AccrueInterst` is called for each money market

Reply:
Added comment in https://github.com/Kava-Labs/kava/pull/720/commits/a8eeb7a26493493293a4dfa7b73f6765c07691c0